python/dist2wheel.py

```diff
@@ -56,7 +56,7 @@ def __init__(self):
         with open("README.md") as f:
             self.readme = f.read()
 
-        self.distribution = self.metadata["project_name"]
+        self.distribution = f"python-{self.metadata['project_name']}"
```

Comment:
It would be wise to double-check that the result files here are still installable, calling the package `go-gilt` seemed to piss it off; this might do the same

Reply:
Yeah, I wont merge any of this without you verifying my hacks